openreview/conference/builder.py

```diff
@@ -83,9 +83,14 @@ def __set_expertise_selection_page(self):
             return self.webfield_builder.set_expertise_selection_page(self, expertise_selection_invitation)
 
     def __set_bid_page(self):
```

Comment:
For clarity, I suggest either: 

(1) changing the name of this function to `__set_bid_pages` (plural) and add a docstring that explains that this function sets the bid pages of both the reviewers page and the AC page (if applicable). 

or (2): parameterizing this function with the group ID, and calling it separately for reviewers and ACs wherever it is called.

I'm not sure if there are other page-setting functions that follow this pattern, but in my opinion it would be clearer one of these two ways

Reply:
I'm not sure if need to change the name to plural, I want to hide the implementation. The method that set a page per group id is in the invitation builder: https://github.com/iesl/openreview-py/blob/fix/improve-bid-webfield/openreview/conference/webfield.py#L120. 

I added a docstring to the function.